src/components/ContentSidebar/ActivityFeed/activity-feed/ActivityFeed.js

```diff
@@ -35,10 +39,18 @@ type Props = {
     feedItems?: FeedItems,
 };
 
+type Props = {
+    logger: LoggerType,
+} & PropsWithoutContext;
+
 type State = {
     isInputOpen: boolean,
 };
 
+const ACTIVITY_FEED_COMPONENT = 'activity_feed';
```

Comment:
Would these fit in the logger constants you defined in a separate file?

Reply:
Sure would!